### PR TITLE
added learn more link to custom liquid

### PR DESCRIPTION
**Pull request by lougoncharenko**
### PR Summary: 
Custom Liquid section info contains a learn more link.

### Why are these changes introduced?

Fixes #2726 .

### What approach did you take?
I added a learn more tag that links to an informational page about liquid

### Visual impact on existing themes
N/A

### Testing steps/scenarios
- [ ] Add a custom liquid section
- [ ] Click on the link in the info paragraph

### Demo links

- [Custom Liquid Section](https://admin.shopify.com/store/os2-demo/themes/140224233494/editor?block=template--17304093458454__3f524a1f-7daa-4033-9325-6f283fece991%2F55746cf6-77b5-4bfc-ba45-f32da95a1eb4&section=template--17304093458454__3f524a1f-7daa-4033-9325-6f283fece991)
- [Editor](https://admin.shopify.com/store/os2-demo/themes/140224233494/editor)

### Checklist
- [ ] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [ ] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [ ] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [ ] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [ ] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [ ] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [ ] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [ ] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Review by ludoboludo — APPROVED** [review]
Looking good 👍 There is something that's part of that type of setting which links to the same docs already. 

Though I don't know if it's an issue 🤔

**Comment by katycobb** [comment]
> Looking good 👍 There is something that's part of that type of setting which links to the same docs already.
> 
> Though I don't know if it's an issue 🤔
> 
> 

Oh good callout, I do think that's an issue. Can we remove that offsite icon?

File: locales/en.default.schema.json [hunk 1]
```diff
@@ -658,7 +658,7 @@
       "settings": {
         "custom_liquid": {
           "label": "Liquid code",
-          "info": "Add app snippets or other code to create advanced customizations."
+          "info": "Add app snippets or other code to create advanced customizations. [Learn more](https://shopify.dev/docs/api/liquid)"
```
Comment (ludoboludo) [inline locales/en.default.schema.json:661]: The icon isn't defined at the theme level 🤔 So we would need to work with the folks that do editor work. Maybe @kmeleta would know 😅

Comment (lougoncharenko) [reply locales/en.default.schema.json:661]: The offsite icon was introduced with the online-store-editor and can't be changed in dawn. We would have to create a separate issue/ticket for this @kimberlyoleiro

**Pull request merged**